### PR TITLE
adding share link and edit button

### PR DESCRIPTION
**Pull request by cxjohn**

### CODE REVIEW

File: lib/components/PollCardControls.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { Poll } from "@prisma/client";
+import { useCopyToClipboard } from "../useCopyToClipboard";
+import EditIcon from "./icons/EditIcon";
+import ShareIcon from "./icons/ShareIcon";
+import CheckIcon from "./icons/CheckIcon";
+
+export default function PollCardControls({ poll }: { poll: Poll }) {
+  const { copied, copyToClipboard } = useCopyToClipboard();
+
+  const pollPath = `/${poll.uid}` + `/${poll.urlSlug}`;
```
Comment (joelburget) [inline lib/components/PollCardControls.tsx:14]: I think `@/lib/links:pollUrl` should do this for you

File: lib/components/PollCardControls.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import Link from "next/link";
+
+import { Poll } from "@prisma/client";
+import { useCopyToClipboard } from "../useCopyToClipboard";
```
Comment (joelburget) [inline lib/components/PollCardControls.tsx:6]: Slight preference for "@/lib/useCopyToClipboard"

File: lib/components/edit-poll/Statements.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ export default function PollStatements({
       />
       <div className="ml-auto mt-6">
         <Button
-          title="Publish poll"
+          title={poll.published ? "Update poll" : "Publish poll"}
           onClick={async () => {
             await publishPoll(poll.uid, statements);
             router.push(`/${poll.uid}/${poll.urlSlug}`);
```
Comment (joelburget) [inline lib/components/edit-poll/Statements.tsx:39]: Aside: We should migrate this to `pollUrl` also.

**Review by joelburget — APPROVED** [review]

File: lib/components/edit-poll/Statements.tsx (hunk 3 above)
Comment (cxjohn) [reply lib/components/edit-poll/Statements.tsx]: I made this update but am unable to test it due to the Anthropic key issue. Please test it if you can!

**Pull request merged**